src/components/GameInfoV2/index.tsx

```diff
@@ -119,13 +134,15 @@ const GameInfoV2: React.FC<GameInfoV2Props> = ({
                       {i18n.earlyAccess}
                     </Sticker>
                   ) : null,
-                  <Sticker
-                    key={i18n.developer}
-                    styleType="neutral"
-                    variant="filledStrong"
-                  >
-                    {info.developer}
-                  </Sticker>,
+                  info.developer ? (
```

Comment:
Same here, it is possible for it to not have developer?

Reply:
same answer as above